spec/unit/artifactserver_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'spec_helper'
 require 'cookbook-omnifetch/artifactserver'
+require 'zlib'
+require 'archive/tar/minitar'
```

Comment:
So far it can't, and I'm 50/50 on whether it's actually a reasonable feature - I almost feel like it'd mostly get built to help testing (which might be ok).